default-config.kdl.nix

```diff
@@ -40,7 +40,6 @@ inputs: # This is your flake inputs. It contains the `niri` flake (github:sodibo
 
 # This is a home-manager module.
 {
-  pkgs,
```

Comment:
In general, this file isn't meant to be touched or used directly really - it exists to show off how one might use the kdl-in-nix DSL provided by this flake.